### PR TITLE
Support using custom extensions of separation files with mina-entry-webpack-plugin

### PR DESCRIPTION
**Pull request by imyelo**

### CODE REVIEW

File: packages/mina-entry-webpack-plugin/lib/loaders/virtual-mina-loader.js [hunk 1]
```diff
@@ -58,17 +53,26 @@ module.exports = function() {
...
+      return fs.exists(filePath).then(isExist => ({ isExist, filePath }))
+    }
+    return pAny(extnames.map(findFileWithExtname), {
+      filter: ({ isExist }) => isExist,
+    }).then(
+      ({ filePath }) => {
+        this.addDependency(filePath)
+        return fs.readFile(filePath, 'utf8')
+      },
+      () => {}
```
Comment (mrbone) [inline packages/mina-entry-webpack-plugin/lib/loaders/virtual-mina-loader.js]: 这里是为了把 error 吞掉？

File: packages/mina-entry-webpack-plugin/package.json [hunk 2]
```diff
@@ -39,7 +39,9 @@
     "fs-extra": "^7.0.1",
     "json5": "^2.1.0",
     "loader-utils": "^1.2.3",
+    "lodash": "^4.17.11",
```
Comment (mrbone) [inline packages/mina-entry-webpack-plugin/package.json]: 这里有必要把整个 Loadsh 作为依赖吗？loader.merge 应该够了吧？

Comment (imyelo) [reply packages/mina-entry-webpack-plugin/package.json]: 考虑跟 mina-loader 保持一致，所以引入了整个 lodash

File: packages/mina-entry-webpack-plugin/lib/loaders/virtual-mina-loader.js (hunk 1 above)
Comment (imyelo) [reply packages/mina-entry-webpack-plugin/lib/loaders/virtual-mina-loader.js]: 对，吞掉 p-any 在找不到文件时抛出的 [AggregateError](https://github.com/sindresorhus/aggregate-error)

**Comment by imyelo** [comment]
released [@tinajs/mina-entry-webpack-plugin@1.2.0](https://github.com/tinajs/mina-webpack/releases/tag/%40tinajs%2Fmina-entry-webpack-plugin%401.2.0)

**Pull request merged**